### PR TITLE
Conda environment refactoring

### PR DESCRIPTION
**Pull request by xmnlab**
This PR is a refactoring for the conda environment.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,28 +1,26 @@
-django==3.1.6
+celery
+dbfread
+dj-database-url
+dj-static
 django-bootstrap4
+django-chunked-upload==1.1.3
 django-leaflet
+django-makemessages-xgettext
+django==3.*
```
Comment (esloch) [inline requirements.txt:9]: Here.

File: environment-dev.yml [hunk 2]
```diff
@@ -1,52 +1,50 @@
...
 channels:
   - conda-forge
-  - defaults
 dependencies:
-- django=3.0.3
+- celery
+- dbfread
+- dj-database-url
+- dj-static
+- django 3.*
```
Comment (esloch) [inline environment-dev.yml:9]: Same solves django from 3.1.6 to 3.1.8.

File: environment-3.8.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+name: alertadengue
+channels:
+  - conda-forge
+dependencies:
+- celery
+- dbfread
+- dj-database-url
+- dj-static
+- django 3.*
```
Comment (esloch) [inline environment-3.8.yml:9]: We have a vulnerability in the current version of django. See #418; It solves with 3.1.8 version.
You can trigger a rebase manually by commenting @dependabot rebase.

File: Makefile [hunk 4]
```diff
@@ -58,7 +58,7 @@ run_staging_db:
 	$(staging_compose_cmd) run --rm staging_db postgres -V
 
 ## Migrate databases and create shapefiles to synchronize with static_files
-build_migrate_staging: run_staging_db
+migrate_staging: run_staging_db
```
Comment (esloch) [inline Makefile:61]: It would be nice to put migrate_staging in the deploy_staging:
```
deploy_staging: migrate_staging
```

Comment (esloch) [reply Makefile:61]: Thanks for contributing @xmnlab!

Comment (fccoelho) [reply Makefile:61]: Thanks @xmnlab !

**Comment by xmnlab** [comment]
@dependabot rebase

**Pull request merged**